uat/testing-features/pom.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Wouldnt you need a parent pom if you want to do the same as otf? 

I want to have some sort of consistency between our repos (so if we go ahead with this we would also want to update log manager) - What custom components are we placing? I guess tu simulate clients updating the shadow?

Reply:
`uat` module is this PR is the parent packaging as a pom without any artifacts. Almost all shadow manager UATs need custom components that perform IPC operations.